.github/workflows/uc-oss.yml

```
@@ -0,0 +1,92 @@
+# .github/workflows/uc.yml
+name: uc-oss
+
+on:
+  pull_request:  # for testing
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
```

```suggestion
      - ready_for_review
    paths:
      - <pattern>
```

we can `paths` to run this workflow when the matched files changed. Not sure if this is necessary.

---

I'll update this to include them, but I figured if we were just running this every night, it wouldnt matter?